lapis-e2e/test/alignedNucleotideSequence.spec.ts

```diff
@@ -143,5 +169,64 @@ describe('The /alignedNucleotideSequence endpoint', () => {
         s: undefined,
       });
     });
+
+    it('should ignore the fasta header template when returning JSON', async () => {
+      const result = await lapisMultiSegmentedSequenceController.postAlignedNucleotideSequence({
+        nucleotideSequenceRequest: {
+          dataFormat: 'JSON',
+          fastaHeaderTemplate: '{primaryKey}{date}{something invalid}',
+        },
+        segment: 'M',
+      });
+
+      expect(result).to.have.length(6);
+      expect(
+        Object.entries(result[0])
+          .filter(([_, value]) => value !== undefined)
+          .map(([key]) => key)
+      ).to.have.members(['primaryKey', 'm']);
+    });
+
+    it('should ignore the fasta header template when returning all sequences JSON', async () => {
+      const result = await lapisMultiSegmentedSequenceController.postAllAlignedNucleotideSequences({
```

Comment:
shouldn't this raise an error if the `fastaHeaderTemplate` contains invalid metadata fields?

Reply:
The purpose of the test is to show that the template is ignored when you do not request fasta format.

Reply:
I changed it. Now it throws an error.